sdcm/cluster.py

```diff
@@ -1243,6 +1246,7 @@ def search_database_log(self, search_pattern=None, start_from_beginning=False, p
                             cloned_event = event.clone_with_info(node=self, line_number=index, line=line)
                             backtraces.append(dict(event=cloned_event, backtrace=[]))
                             matches.append((index, line))
+                            break
```

Comment:
please add comment here, why you are breaking

Reply:
@dkropachev why did you add a break?

Reply:
I can assume the reason, to make sure each line has only one match. otherwise adding a new type to the list won't help, cause it would be still matched by other regexps. but should have a comment explaining exactly that